flake-info/src/bin/flake-info.rs

```diff
@@ -404,3 +420,30 @@ async fn push_to_elastic(
 
     Ok(())
 }
+
+/// Information about allowed and default nixos channels.
+/// Typyically passed by environment variable NIXOS_CHANNELS.
+/// Used to filter the input arguments for `flake-info nixpkgs` and `flake-info nixpkgs-archive`
```

Comment:
Not sure I understand the point of checking this in flake-info. What if I want to run `nixpkgs-archive` locally on some random branch? Will that fail?